pkg/serdes/protobuf_serialization_provider.go

```diff
@@ -108,6 +149,34 @@ func parseMessage(schemaPath string, referencePathMap map[string]string) (gproto
 		ImportPaths: importPaths,
 	}
 
+	currDir, err := os.Getwd()
+	if err != nil {
+		return nil, fmt.Errorf("Error getting current working directory: %v\n", err)
+	}
+	log.CliLogger.Debugf("Current working directory is: %s\n", currDir)
+
+	// Copy all the built-in proto schemas needed for CSFLE to <importPath> where the main schema is stored
+	// Note: folder path should be set correctly based on current working directory
+	// Expected working directory in CLI test is: /Users/github.com/confluentinc/cli/pkg/serdes
+	// Expected working directory in CLI shell execution is: /Users/github.com/confluentinc/cli
+	for _, folder := range builtInSchemaFoldersToCopy {
+		dst := importPath + "/" + folder
+
+		// Skip copying the built-in schema folders if they are present already in the temp folder
+		if _, err = os.Stat(dst); err == nil {
+			log.CliLogger.Debugf("Built-in schema folder already exists %s, skipping copy again:\n", dst)
+			continue
+		}
+
+		// Locate the source of built-in schema folders
+		if strings.HasSuffix(currDir, "confluentinc/cli") {
+			folder = "pkg/serdes/" + folder
+		}
+		if err = copy.Copy(folder, dst); err != nil {
+			return nil, fmt.Errorf("Error copying built-in schemas folder %s: %w\n", folder, err)
+		}
```

Comment:
@sgagniere This part is something relatively new to CLI, your review will be appreciated!